bundles/org.openhab.binding.solarmax/README.md

```diff
@@ -0,0 +1,58 @@
+# SolarMax Binding
+
+This binding supports SolarMax PV inverters.
+
+## Supported Things
+
+This binding only has a single `inverter` thing that can be added manually.
+The SolarMax MT Series is supported (tested with 8MT2 devices).
+
+## Discovery
+
+Auto-discovery is currently not available.
+
+## Thing Configuration
+
+Each inverter requires the following configuration parameters:
+
+| parameter       | required | default | description                                                          |
+| --------------- | -------- | ------- | -------------------------------------------------------------------- |
+| host            | yes      |         | hostname or IP address of the inverter                               |
+| port            | no       | 12345   | Port number to connect to. This should be `12345` for most inverters |
+| refreshInterval | no       | 15      | Interval (in seconds) to refresh the channel values.                 |
+
+
+## Channels
+
+| channel                  | type     | description                                            |
+| ------------------------ | -------- | ------------------------------------------------------ |
+| lastUpdated              | DateTime | Time when data was last read from the device           |
```

Comment:
The type must be for example `Number:Temperature`.

Reply:
Updated